ignite/handlers/state_param_scheduler.py

```diff
@@ -13,6 +15,9 @@ class StateParamScheduler(BaseParamScheduler):
         param_name: name of parameter to update.
         save_history: whether to log the parameter values to
             `engine.state.param_history`, (default=False).
+        create_new: in case `param_name` already exists in `engine.state`, whether to authorize `StateParamScheduler`
```

Comment:
@fco-dv please use double-ticks ```` instead of single-tick when addresses code. Here and above.

```suggestion
        create_new: in case ``param_name`` already exists in ``engine.state``, whether to authorize ``StateParamScheduler``
```